copy_properties.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Can this just get used locally? Might change docs if so

Reply:
Didn't consider that but yeah absolutely:
`./copy_properties.sh 'common/src/main/resources/properties/'`
I would add one warning though: this will overwrite any existing `.properties` files that already exist